<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DEditUser</class>
 <widget class="QWidget" name="DEditUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(0, 0, 0, 120);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>52</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>52</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>287</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>287</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>780</width>
       <height>380</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">width: 630px;
height: 510px;
background: #FFFFFF;
border-radius: 20px;</string>
     </property>
     <widget class="QLabel" name="lab_title">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;

width: 46px;
height: 23px;
font-family: PingFang SC;
font-weight: 500;
font-size: 24px;
color: #ffffff;
line-height: 34px;</string>
      </property>
      <property name="text">
       <string>修改用户</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>100</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 22px;
font-family: &quot;PingFang SC&quot;;
color: #666666;
</string>
      </property>
      <property name="text">
       <string>账号名</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_userId">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>90</y>
        <width>300</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #FAFBFC;
border: 1px solid #DCDEE0;
border-radius: 6px;
font-size: 18px;
font-family: &quot;PingFang SC&quot;;
padding-left: 10px;</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhNone</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_userType">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>140</y>
        <width>300</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox{
	border: 1px solid #DCDEE0;
	border-radius: 6px;
	font-family: &quot;PingFang SC Medium&quot;;
	font-size: 18px;
	padding-left: 10px;
	background-color: rgb(250, 251, 252);
	color: #333333;
}
QComboBox::drop-down{
	margin: 0px;
	padding-right: 20px;
	border: 0px;
}
QComboBox::down-arrow{
	image: url(:/STATIC/PHOTOS/combo_down.png);
}
QComboBox::down-arrow:on{
	image: url(:/STATIC/PHOTOS/combo_up.png);
}
QListView{
	margin: 0px;
	padding: 0px;
	border: 0px;
	border-radius: 10px;
	outline: 0px;
	font-size: 20px;
	color: #333333;
	background-color: rgb(250, 251, 252);
}
QListView::item{
	margin: 0px;
	padding: 10px;
	border: 0px;
	border-radius: 0px;
	height: 30px;
}
QListView::item:hover{
	color: #333333;
}
QListView::item:selected{
	color: rgb(31, 112, 227);
}</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>150</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 22px;
font-family: &quot;PingFang SC&quot;;
color: #666666;
</string>
      </property>
      <property name="text">
       <string>账号类型</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>200</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 22px;
font-family: &quot;PingFang SC&quot;;
color: #666666;
</string>
      </property>
      <property name="text">
       <string>新密码</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_pdNew">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>190</y>
        <width>300</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #FAFBFC;
border: 1px solid #DCDEE0;
border-radius: 6px;
font-size: 18px;
font-family: &quot;PingFang SC&quot;;
padding-left: 10px;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_no">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>280</y>
        <width>200</width>
        <height>80</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>1111</width>
        <height>1111</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: transparent;
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(244, 96, 117, 255), stop:1 rgba(201, 67, 86, 255));
border-radius: 10px;
font-weight: bold;
color: white;
font-size: 32px;
font-family: PingFang SC;</string>
      </property>
      <property name="text">
       <string>取消</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_yes">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>280</y>
        <width>200</width>
        <height>80</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>1111</width>
        <height>1111</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background: #22A4B3;
border-radius: 10px;

font-family: PingFang SC;
font-weight: bold;
font-size: 32px;
color: #FFFFFF;
line-height: 42px;</string>
      </property>
      <property name="text">
       <string>确认</string>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="widget_5" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>780</width>
        <height>60</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>480</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
background: none;
image: none;

background: #22a4b3;

border-top-left-radius: 20px;
border-top-right-radius: 20px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
      </property>
     </widget>
     <zorder>widget_5</zorder>
     <zorder>lab_title</zorder>
     <zorder>label_2</zorder>
     <zorder>line_userId</zorder>
     <zorder>combo_userType</zorder>
     <zorder>label_3</zorder>
     <zorder>label_6</zorder>
     <zorder>line_pdNew</zorder>
     <zorder>btn_no</zorder>
     <zorder>btn_yes</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
